components/access-management/kfam/bindings.go

```diff
@@ -184,28 +184,35 @@ func (c *BindingClient) List(user string, namespaces []string, role string) (*Bi
 			return nil, err
 		}
 		for _, roleBinding := range roleBindings {
-			userVal, ok := roleBinding.Annotations[USER]
-			if !ok {
-				continue
+			var subject rbacv1.Subject
+			for _, roleBindingSubject := range roleBinding.Subjects {
+				if roleBindingSubject.Kind == USER && roleBindingSubject.Name == user {
+					subject = roleBindingSubject
+					break
+				}
 			}
-			if user != "" && user != userVal {
-				continue
+			if subject == (rbacv1.Subject{}) {
+				userVal, ok := roleBinding.Annotations[USER]
+				if !ok {
+					continue
+				}
+				if user != "" && user != userVal {
```

Comment:
Why are checking if the user is equal to the annotation?

Reply:
It'll be skipped if the `Subject` check is successful, but the idea is for the new checks to be additive and the change is fully backwards compatible.

Reply:
cool